src/axom/mir/docs/sphinx/mir_utilities.rst

```diff
@@ -224,13 +325,22 @@ merge points.
    :end-before: _mir_utilities_unique_end
    :language: C++
 
+##################
+VariableShape
+##################
+
+The ``axom::mir::utilities::blueprint::VariableShape`` class behaves like a primal shape
+but it can represent various 3D shapes, some not present in primal.
```

Comment:
Should we add more primal primitives?

Reply:
Longer term, it would be good to add Pyramid and Wedge classes. In this case, VariableShape is used to do 2 things: 1) implement some pyramid/wedge functionality, 2) act as a "variable" shape that can act like a tet,pyramid,wedge,hex shape for when we get "mixed" unstructured meshes from Blueprint. Moving to Polyhedron for that case was overkill.